modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -75,6 +77,11 @@ public String getLocalizationDBTableName() {
 		return getDBTableName() + "_l";
 	}
 
+	@Override
+	public List<ObjectAction> getObjectActions() {
```

Comment:
I like the approach you used for `_defaultObjectLayouts`, so I think we can narrow down the results here by only getting the information we need. So change that to `_standaloneObjectActions` and modify the finder to look for those related to that type that are active.

Reply:
Just done!